tensorflow_datasets/core/dataset_builder.py

```diff
@@ -28,11 +28,16 @@
 
 from absl import logging
 import six
-import tensorflow.compat.v2 as tf
+import tensorflow.compat.v2 as tf  # type: ignore
+import apache_beam as beam
```

Comment:
Do not import beam directly, as it it would make TFDS import raise error for all users which do not have installed Beam.

Instead, you can use:

```
if typing.TYPE_CHECKING:
  try:
    import apache_beam as beam
  except ImportError:
    beam = Any
```

Reply:
Done